src/main/java/io/castle/example/SetupListener.java

```diff
@@ -0,0 +1,28 @@
+package io.castle.example;
+
+import io.castle.client.Castle;
+import io.castle.client.model.CastleSdkConfigurationException;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+
+public class SetupListener implements ServletContextListener {
+
+    /**
+     * Example of how to verify the SDK configuration during application initialization to avoid errors during request handling.
+     *
+     * @param sce
+     */
+    public void contextInitialized(ServletContextEvent sce) {
+        try {
+            Castle.verifySdkConfigurationAndInitialize();
```

Comment:
could you extend the example to changing something in the configuration. e.g. add a header to the blacklist

Reply:
The configuration file contains additional headers for the blacklist, please check the file:
src/main/resources/castle_sdk.properties